waflib/Scripting.py

```diff
@@ -264,7 +264,7 @@ def run_commands():
 	while Options.commands:
 		cmd_name = Options.commands.pop(0)
 		ctx = run_command(cmd_name)
-		Logs.info('%r finished successfully (%s)', cmd_name, ctx.log_timer)
+		Logs.debug('%r finished successfully (%s)', cmd_name, ctx.log_timer)
```

Comment:
I don't like this change, I search for this string a lot when looking at CI logs. Also, it says the time it took, which is very useful.

Reply:
if we want the time it took we can simply prefix the command with "time". If only it went to stderr instead of stdout, it would be less annoying. Right now I type "./waf board" or "./waf list_boards" and have that output there.

Reply:
That's not the same, this gives the time for each command. But that's not even my main reason, I like this message and I use it, I don't understand what's so annoying about it.

Reply:
It's a debug message for whom is looking to the build system, not anybody else. If you look at our scripts you will see we do a `|  head -n 1` to parse the output of our commands just because waf decided it should inform how much time each and every task took. No other build system on earth do that by default. If they do it's always an opt-in. Here we don't even have an opt-out.

Reply:
> That's your opinion... It's informative to me.

Of course it is. We are all biased to our own use and opinions.

> We do that in one place... And yes, that's a decision that waf took. Just because others don't do it doesn't mean anything.

It does... probably that I'm not alone with it and I doubt the majority of users care about it.

> If you want an opt-out option that's fine, it would probably even be accepted upstream.

nah. If I'm proven wrong that more devs want this, then I can simply don't bother changing anything... I will just never remember what to type to remove it, so it will just be another annoyance. For the usecase you presented on the other hand, it could be added the option to the build script.